lib/cli/interactive-setup/dashboard-login.js

```diff
@@ -51,7 +52,19 @@ module.exports = {
     }
 
     const sdk = new ServerlessSDK();
-    const { supportedRegions, supportedRuntimes } = await sdk.metadata.get();
+    const sdkMetadata = await (async () => {
```

Comment:
This change should not be part of this PR

Reply:
Fixed